client/src/components/template/templateCard.component.js

```diff
@@ -17,15 +17,15 @@ cardIconDictionary.add("Hardware Training", "build");
 cardIconDictionary.add("Soft Skills Training", "group");
 cardIconDictionary.add("General Training", "work");
 
-const TemplateDetails = props => {
+const TemplateDetails = (props) => {
   useEffect(() => {
     let elems = document.querySelectorAll(".modal");
     M.Modal.init(elems, {});
   }, []);
 
-  const deleteThis = e => {
+  const deleteThis = (e) => {
     e.preventDefault();
-    axios.post(`/inviteTemplate/delete/${props.item._id}`).then(res => {
+    axios.post(`/inviteTemplate/delete/${props.item._id}`).then((res) => {
       console.log(res.data);
```

Comment:
if everything works fine then I think you can remove consol logos

Reply:
Done.